### ISSUE TITLE
"That's" is not recognized as contraction

### ISSUE THREAD

**Issue by loganwilliams** [nlp-compromise/compromise#322]
"That's" does not seem to be recognized as a contraction, which is screwing up the ability to tag people in some simple examples I was playing around with.

```
> var p = nlp('That\'s Evan.');
> p.contractions().data()
[]
> p.people().data()
[]
> var p = nlp('Oh, that\'s John Edwards.');
> p.people().data()
[ { text: ' Edwards.',
    normal: 'edwards',
    firstName: '',
    middleName: '',
    lastName: 'edwards.',
    genderGuess: null,
    pronoun: 'they',
    honorifics: [] } ]
> p.contractions().data()
[]
> var p = nlp('That is Evan.');
> p.contractions().data()
[]
> p.people().data()
[ { text: ' Evan.',
    normal: 'evan',
    firstName: 'evan.',
    middleName: '',
    lastName: '',
    genderGuess: 'Male',
    pronoun: 'he',
    honorifics: [] } ]
```

**Comment by spencermountain** [comment]
thanks Logan, i'll take a look

**Comment by spencermountain** [comment]
fixed in `v8.1.0`
thanks.
